src/status_im2/contexts/chat/lightbox/text_sheet/utils.cljs

```diff
@@ -68,5 +82,5 @@
 
 (defn init-derived-animations
   [{:keys [derived-value]}]
-  {:height (worklet/text-sheet derived-value true)
-   :top    (worklet/text-sheet derived-value false)})
+  {:height derived-value
+   :top    (reanimated/interpolate derived-value [0 1] [0 -1])})
```

Comment:
Are you interpolating the `top` value between `0` and `-1` ?

Reply:
I'm using interpolation to negate the `derived-value`. Since the interpolation is not clamped, it will be applied to the entire range of values.